pyquaticus/envs/pyquaticus.py

```diff
@@ -1045,6 +1050,18 @@ def set_config_values(self, config_dict):
 
         self.num_renders_per_step = int(self.render_fps * self.tau)
 
+        # check that time warp is a integer >= 1 and only active if rendering is on
+        if self.sim_speedup_factor < 1:
+            print("Warning: sim_speedup_factor must be an integer >= 1! Defaulting to 1.")
+            self.sim_speedup_factor = 1
+
+        if type(self.sim_speedup_factor) != int:
```

Comment:
```suggestion
        if not isinstance(self.sim_speedup_factor, int):
```

Reply:
Using `isinstance` is a little faster and supports inheritance (even though that's not needed here) so that's the more pythonic way to check this. If you do keep the direct `type` comparison, it would be better to use `is not` instead of ` !=` for both speed and pythonic style.